cocos2d/Support/CCFile_Private.h

```diff
@@ -55,6 +55,13 @@ typedef NSInputStream *(^CCStreamedImageSourceStreamBlock)();
 @end
 
 
+@interface CCFile()
+
+@property (nonatomic, assign, readwrite) BOOL useUIScale;
+
+@end
+
+
 @interface CCFile(Private)
```

Comment:
That was the first thing I did but It didn't work so I moved it in a class extension.